master/buildbot/worker/docker.py

```diff
@@ -148,7 +147,8 @@ def reconfigService(self, name, password, docker_host, image=None, command=None,
             self.client_args['tls'] = tls
 
     def parse_volumes(self, volumes):
-        self.volumes = []
+        volume_list = []
```

Comment:
Please rename this function _thd_parse_volumes() to tell that this method must take care of thread safety